File: src/manual_process/src/pcl_tutorial/src/point_cloud_subscriber.py
#!/usr/bin/env python

import rospy
#import pcl
from sensor_msgs.msg import PointCloud2
#import sensor_msgs.point_cloud2 as pc2 
#import ros_numpy

def raw_points_cb(msg):
    print msg.fields
    print "\n"


if __name__ == "__main__":
    rospy.init_node('point_cloud_data', anonymous=True)
    rospy.Subscriber('camera/depth/points', PointCloud2, raw_points_cb)
    rospy.spin()

File: src/manual_process/src/pcl_tutorial/src/point_cloud_subscriber.cpp
#include "ros/ros.h"
#include "sensor_msgs/PointCloud2.h"

void point_cloud_callback(const sensor_msgs::PointCloud2::ConstPtr& msg){
    ROS_INFO("Height is :[%d]", msg->height); 
    ROS_INFO("Width is: [%u]", msg->width);
    

int main(int argc, char **argv){
    ros::init(argc, argv, "point_cloud_sub"); 
    ros::NodeHandle nh; 
    ros::Subscriber sub = nh.subscribe("camera/depth/points", 1000, point_cloud_callback);
    ros::spin();
    return 0;
}